README.md

```diff
@@ -58,7 +58,8 @@ If you're looking for **Controller models**:
     * ControllerInfo.cs
     * GLTFComponentStreamingAssets.cs
     * The entire Utilities\Scripts\GLTF folder.
-* **IMPORTANT** You will also need a recent Windows Insider SDK installed, or you will not be able to build these scripts. Please install the SDK found [here](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK).
+* **IMPORTANT** You need to have the 16299 UWP SDK installed, or you will not be able to build these scripts. Please install the SDK found [here](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk).
```

Comment:
You can download the SDK directly via Visual Studio Installer.

I also added this down in the [prerequisites](https://github.com/StephenHodgson/MixedRealityToolkit-Unity/blob/MRTK-DevMerge/README.md#prerequisites).

Reply:
That's good. The goal of changing this line is because the info is outdated as of 6 days ago (DevDay). Updating it will prevent the reader from downloading the wrong one when MS releases the next preview SDK or thinking they'll need insider bits to proceed.